Packs/CiscoUmbrellaReporting/Integrations/CiscoUmbrellaReporting/CiscoUmbrellaReporting_test.py

```diff
@@ -0,0 +1,802 @@
+            expected output.
+    """
+    mocker.patch.object(client, 'query', side_effect=[raw_response])
+    args = {
+        "page_size": 5,
+        "from": 1662422400000,
+        "to": 1662768000000,
+        "page": 2,
+    }
+    page_2 = PAGE_2_PAGE_SIZE_5_LIST
+    command_results_new = get_activity_list_command(client, args)
+    # results is CommandResults list
+    next_page = command_results_new.to_context()['Contents']
+    assert next_page[5:10] == page_2.get("data")
+
+
+def test_pagination():
+    """
+    Tests the pagination function.
+
+        Given:
+            no argument required
+
+        When:
+            - Running the 'pagination function'.
+
+        Then:
+            -  Checks the output of the command function with the
+            expected output.
+    """
+    page = 2
+    page_size = 5
+    expected_response = (5, 5)
+
+    context_detail = pagination(page, page_size)
+    assert context_detail == expected_response
+
+
+@pytest.mark.parametrize('raw_response', [DESTINATION_LIST_RESPONSE])
+def test_test_module(requests_mock, raw_response):
+    """
+        Tests the test_module function.
+
+            Given:
+                no argument required
+
+            When:
+                - Running the 'test_module function'.
+
+            Then:
+                -  Check weather the given credentials are correct or not
+    """
+    from CiscoUmbrellaReporting import test_module
+    post_req_url = client.token_url
+    requests_mock.post(post_req_url, json={'access_token': '12345'})
+    access_token = client.access_token()
+    get_req_url = f'{client._base_url}/v2/organizations' \
+                  f'/{client.organisation_id}/activity'
+    headers = {'Authorization': f'Bearer {access_token}'}
+    requests_mock.get(get_req_url, headers=headers, json=raw_response)
+    output = test_module(client)
+    assert output == 'ok'
+    with pytest.raises(DemistoException):
+        error_output = {
+            "meta": {},
+            "data": {
+                "error": "unauthorized"
+            }
+        }
+        requests_mock.get(get_req_url, headers=headers, status_code=401, json=error_output)
+        test_module(client)
+
+
+def test_access_token(requests_mock):
+    """
+        Tests the access_token function.
+
+            Given:
+                - requests_mock object.
+
+            When:
+                - Running the 'get_activity_list_command function'.
+
+            Then:
+                -  Checks the output of the command function with the
+                expected output.
+        """
+    req_url = client.token_url
+
+    requests_mock.post(req_url, json={'access_token': '12345'})
+
+    response = client.access_token()
+    assert response == "12345"
+    with pytest.raises(DemistoException):
+        requests_mock.post(req_url, status_code=401)
+        client.access_token()
+        requests_mock.post(req_url, status_code=400)
+        client.access_token()
```

Comment:
Please add a test for each auxiliary function - I think that the following don't have tests: 
check_valid_indicator_value
get_command_title_string
activity_build_data
create_cisco_umbrella_args

Reply:
@ShacharKidor Added